app/views/admin/layouts/_area.html.erb

```diff
@@ -0,0 +1,12 @@
+<% x = area.coordinate.nil? ? 0 : area.coordinate.x %>
+<% y = area.coordinate.nil? ? 0 : area.coordinate.y %>
+<% type = area.class.name.downcase %>
+<div class="<%= type %> draggable"
+     data-toggle="tooltip"
+     title="<%= area.name %>"
+     style="transform: translate(<%= x %>px, <%= y %>px)"
+     rails-id="<%= area.id %>"
```

Comment:
Should we go with more standard attribute names? `data-id` and `data-type`?

Reply:
Good idea. Will do